tests/modules/nf-scil/tracking/localtracking/nextflow.config

```diff
@@ -0,0 +1,19 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
```

Comment:
Put publishdir into the module parameters : 

withName: "MODULE" {
     publishdir =  { ...}